lib/test_prof/recipes/rspec/before_all.rb

```diff
@@ -9,26 +9,25 @@ module RSpec
       def before_all(&block)
         raise ArgumentError, "Block is required!" unless block_given?
 
-        return within_before_all(&block) if within_before_all?
+        return before(:all, &block) if within_before_all?
 
         @__before_all_activated__ = true
 
         before(:all) do
-          BeforeAll.begin_transaction do
-            instance_eval(&block)
-          end
+          BeforeAll.begin_transaction
+          instance_eval(&block)
         end
 
         after(:all) do
           BeforeAll.rollback_transaction
         end
-      end
 
-      def within_before_all(&block)
```

Comment:
This boils down to `before(:all, &block)` since `within_transaction` was a passthrough. I've inlined the only usage.